hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -321,8 +331,13 @@ def _store_state(self, contents: t.Iterable["CliIndicatorSerialization"]):
             write_s3_text(txt_content, self.s3_bucket, self.data_s3_key)
             logger.info("%d rows stored for %d", len(items), self.privacy_group)
 
-    def _apply_updates_impl(self, delta: tu.ThreatUpdatesDelta) -> None:
+    def _apply_updates_impl(
+        self,
+        delta: tu.ThreatUpdatesDelta,
+        post_apply_fn=lambda x: None,
```

Comment:
Can we type the lambda inputs more? I would think that the post_apply function takes a single update but it looks like it takes a dict

Reply:
This is a method override so type granularity is complicated/could go wrong, where and how `_apply_updates_impl` uses the `post_apply_fn` method is dependent on the store class implementing it.